tsconfig.json

```diff
@@ -3,7 +3,7 @@
   "compileOnSave": false,
   "compilerOptions": {
     "outDir": "./dist/out-tsc",
-    "strict": true,
+    "strict": false,
```

Comment:
please set this back to true, if you have errors in the console they probably need to be fixed 😄